### ISSUE TITLE
Symmetry handling in the case of non-uniform orientational distributions

### ISSUE THREAD

**Issue by stestoll** [StollLab/EasySpin#252]
When determining grid symmetry, it is crucial to take the symmetry of the orientational distribution into account. Or just set grid symmetry to `'Ci'` if a non-uniform orientational distribution is specified via `Exp.Ordering`.

```matlab
clf, clear, clc

Sys.g = [2 2 2.3];
Sys.lwpp = 1;

Exp.mwFreq = 9.5;
Exp.Harmonic = 0;
Exp.Ordering = 4;

Opt.GridSymmetry = 'Ci';  % required for correct simulation

beta = linspace(0,pi/2,10);
for k = 1:numel(beta)
  Exp.SampleRotation = {beta(k),'y'};
  [B,spc(:,k)] = pepper(Sys,Exp,Opt);
end

plot(B,spc)
axis tight
```

**Comment by stestoll** [comment]
The grid symmetry needs to be dropped to `'C1'`, not `'Ci'`, since the orientational distribution function can have any symmetry, including no symmetry.

**Comment by stestoll** [comment]
Fixed with f0ec5d39b92cb6374a551722a688f861d58bed0a
